### DISCUSSION TITLE
Blacklist?

### DISCUSSION THREAD

**Discussion by stellayuri** [AprilSylph/XKit-Rewritten#224] [Help Desk]
Hello! I saw a post about the new X-kit rewritten on my dash and I would like to ask a question about it. The predecessor "New Xkit" has a Blacklist feature in which you could add terms like triggers or usernames and it would hide it on your dash. There is also a feature on New Xkit that hides the post entirely rather than showing you the blocked post like "This post was blocked because it contains [Blacklisted term]". Is there a feature like that on Xkit Rewritten or is that not implemented yet? I noticed there was a cleanfeed but when I clicked on it, there was only an option to hide stuff from flagged blogs and all blogs. 

Also thank you for adding a feature that hides the recommendation bar on the side because I was tired of being recommended users on my blacklist / that I've blocked. Thanks for your hard work!

**Comment by AprilSylph** [comment]
https://github.com/AprilSylph/XKit-Rewritten/wiki/Excluded-features#blacklist

**Reply by cathahand** [reply]
Hi @AprilSylph, I appreciate that Tumblr now has a blacklisting feature - but does is have whitelisting like New Xkit? E.g. I have 'spider' blacklisted, but 'spiderman' whitelisted.
Thanks!

**Reply by AprilSylph** [reply]
A feature that integrates a whitelist with Tumblr's filtering is coming, in the form of "Highlighter" (which as you might remember from XKit 7, had an option to auto-unhide posts hidden by Blacklist).

**Reply by cathahand** [reply]
Awesome, thank you!

**Comment by manya-t** [comment]
The other way that the Blacklist function in Xkit was better (for, uh, me personally) than the tumblr filtered tags functionality is that with Blacklist, you had the option to see the tags on blocked posts. Does an option for that sound like something you could add?

**Comment by omniraptorr** [comment]
Tumblr's native block functionality is missing one aspect- if you block someone it hides their posts on your dashboard but not from people's individual blogs. The old blacklist plugin let you hide the latter as well. It's a minor bit of functionality but it is bothering some people so there might be demand for this (visually filtering posts where the op is blocked).

**Reply by AprilSylph** [reply]
Adding a username to Tumblr's keyword filtering works exactly the same as adding it to New XKit's Blacklist.

**Reply by marcustyphoon** [reply]
To be clear: April is saying that if you **filter a username** it'll do this, which is of course not the same thing as **blocking the user**. So yes there's a way to get the "visually filtering posts where the op is blocked" behavior you want, but no, Tumblr's "block" function does not in itself do it.
